<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWidget</class>
 <widget class="QWidget" name="StartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>坦克动荡</string>
  </property>
  <widget class="QPushButton" name="Start2P">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>双人游戏</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="startHelp">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏玩法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GameRecordButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>对局记录</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
